docs/api/spec.md

```diff
@@ -125,7 +127,18 @@ try {
 catch (er){
     console.log(er.message);
 }
+```
 
+```js
+try {
+    const result = await fdc3.raiseContext(context);
+    if (result.data) {
+        const orderId = result.data.id;
+    }
+}
+catch (er){
+    console.log(er.message);
+}
```

Comment:
I suggest expanding `IntentResolution` with an `intentName` property, which will match the provided name when using `raiseIntent`, but will be populated with the selected `intentName` when resolved by the user or desktop agent based on context alone.

Reply:
> I suggest expanding `IntentResolution` with an `intentName` property, which will match the provided name when using `raiseIntent`, but will be populated with the selected `intentName` when resolved by the user or desktop agent based on context alone.

Just to confirm, will consider this as a separate Issue/PR 👍

Reply:
https://github.com/finos/FDC3/issues/284